tests/c/simple_interp_loop2.c

```diff
@@ -12,7 +12,7 @@
 //     yk-jit-event: stop-tracing
 //     --- Begin jit-pre-opt ---
 //     ..~
-//     guard true, %{{34}}, [2:%0_2: %{{0}}, 2:%0_4: %{{1}}, 2:%0_5: %{{2}}, 2:%0_6: %{{3}}, 2:%0_7: %{{4}}, 2:%0_9: %{{5}}, {{_}}:%23_3: %{{34}}]
+//     guard true, %34, [2:%0_{{_}}: %0, 2:%0_{{_}}: %1, 2:%0_{{_}}: %2, 2:%0_{{_}}: %3, 2:%0_{{_}}: %4, 2:%0_{{_}}: %5, 2:%23_{{_}}: %34]
```

Comment:
I think `%34` should stay `%{{34}}` otherwise the test fails if the IR numbers change even slightly.

Reply:
OK. Shall I squash that change straight in?

Reply:
Erp. Found another way the test can fail. Fixed in a0ba4fefebe62fe08c45126401b25dc997c17ce3.